### PR TITLE
Notation: OAuth Consent Screen Publishing Status needs to be In Production, not Testing

### PR DESCRIPTION
**Pull request by danielbrunt57**
## Proposed change

Added notation that OAuth Consent Screen Publishing Status needs to be In Production, not Testing

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by homeassistant** [comment]
Hi @danielbrunt57, 

It seems you haven't yet signed a CLA. Please do so [here](https://home-assistant.io/developers/cla_sign_start/?pr=home-assistant&#x2F;home-assistant.io#16476).

Once you do that we will be able to review and accept this pull request.

Thanks!

File: source/_integrations/nest.markdown [hunk 1]
```diff
@@ -78,7 +78,8 @@ Project**. Note: This is a different type of project from the Device Access proj
 
 1. Give your API Project a name then click **Create**. Note: You can ignore the *Project ID* here as Home Assistant does not need it.
 
-1. Click *OAuth consent screen* and make sure you have that configured, otherwise, you can do that now.
+1. Click *OAuth consent screen* and make sure you have that configured, otherwise you can do that now...
+   - Ensure that the ***Publishing status*** is **In Production** and not **Testing**!!
```
Comment (frenck) [inline source/_integrations/nest.markdown]:
```suggestion
   - Ensure that the ***Publishing status*** is **In Production** and not **Testing**!
```

**Review by frenck — APPROVED** [review]
Thanks, @danielbrunt57 👍

**Comment by allenporter** [comment]
Thank you!

**Pull request merged**